src/rprblender/export/mesh.py

```diff
@@ -322,8 +324,20 @@ def sync(rpr_context: RPRContext, obj: bpy.types.Object, **kwargs):
         rpr_context.create_empty_object(obj_key)
         return
 
+    transform = object.get_transform(obj)
     deformation_data = rpr_context.deformation_cache.get(obj_key)
-    if deformation_data and np.any(data.vertices != deformation_data.vertices) and \
+
+    if smoke_modifier and isinstance(rpr_context, RPRContext2):
+        transform = volume.get_transform(obj)
+        rpr_shape = rpr_context.create_mesh(
+            obj_key,
+            data.vertices, data.normals, data.uvs,
```

Comment:
Do we need to provide mesh data here with MESH_VOLUME_FLAG?

Reply:
Yes, it can be optimized. Fixed with creating an empty Mesh object.